### PR TITLE
add vpc peer connect commit

### PR DESCRIPTION
**Pull request by wjf1993**
**What this PR does / why we need it**:

**Does this PR need to be backport to the previous release branch?**:

### CODE REVIEW

File: pkg/apis/compute/vpc_peer_inwards_const.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	VPC_PEERING_INWARDS_STATUS_UNKNOWN        = "unknown"
+)
+
+type VpcPeeringConnectionInwardsDetails struct {
+	apis.EnabledStatusInfrasResourceBaseDetails
+	VpcResourceInfo
+
+	//SVpcPeerInward
+
+	VpcPeerId string `json:"vpcpeerid"`
```
Comment (ioito) [inline pkg/apis/compute/vpc_peer_inwards_const.go:23]:
```suggestion
	VpcPeerId string `json:"vpc_peer_id"`

```

File: pkg/apis/compute/vpc_peer_inwards_const.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+	VpcLocalId string `json:"vpclocalid"`
```
Comment (ioito) [inline pkg/apis/compute/vpc_peer_inwards_const.go:25]:
```suggestion
	VpcLocalId string `json:"vpc_local_id"`

```

File: pkg/compute/models/vpc_peer_inward.go [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+	db.SEnabledStatusInfrasResourceBaseManager
+	db.SExternalizedResourceBaseManager
+	SVpcResourceBaseManager
+}
+
+type SVpcPeerInward struct {
+	db.SEnabledStatusInfrasResourceBase
+	db.SExternalizedResourceBase
+
+	//new for test
```
Comment (ioito) [inline pkg/compute/models/vpc_peer_inward.go:49]:
```suggestion

```

File: pkg/compute/models/vpc_peer_inward.go [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+	//SManagedResourceBase
```
Comment (ioito) [inline pkg/compute/models/vpc_peer_inward.go:50]:
```suggestion

```

File: pkg/compute/models/vpc_peer_inward.go [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+	SVpcResourceBase
+
+	PendingDeleted bool `nullable:"false" default:"false" index:"true" get:"user" list:"user" json:"pending_deleted"`
+
+	VpcPeerId string `width:"128" charset:"ascii"  list:"user" json:"vpcpeerid"`
```
Comment (ioito) [inline pkg/compute/models/vpc_peer_inward.go:55]:
```suggestion
	VpcPeerId string `width:"128" charset:"ascii" list:"user"`

```

File: pkg/compute/models/vpc_peer_inward.go [hunk 6]
```diff
@@ -0,0 +1,362 @@
+package models
+
+import (
+	"context"
+	"database/sql"
+	"strings"
+
+	"yunion.io/x/cloudmux/pkg/cloudprovider"
```
Comment (zexi) [inline pkg/compute/models/vpc_peer_inward.go:8]: Please reference https://www.cloudpods.org/zh/docs/development/specification/#%E5%AF%BC%E5%85%A5%E6%A0%87%E5%87%86%E5%BA%93%E7%AC%AC%E4%B8%89%E6%96%B9%E6%88%96%E5%85%B6%E5%AE%83%E5%8C%85 to format imported packages

**Pull request closed without merging**